components/organisms/Login/Login.tsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect } from "react";
+
+import * as process from "process";
+
+export type LoginProps = {
+  setSalesforceId: (arg: string) => void;
+  setIsFirst: (arg: boolean) => void;
+  onlyLoginBox?: boolean;
+};
+
+export const Login = ({ setSalesforceId, onlyLoginBox }: LoginProps) => {
```

Comment:
Why

Reply:
I reuse the component when we share queries but i don't want to display the text over again, so i introduce a prop that indicated if we only want the login box and nothing else.

Reply:
like have the login box when a non logged in user looks at a shared query